src/style.css

```diff
@@ -84,74 +92,130 @@ nav {
   font-family: var(--font-roboto);
   font-weight: 700;
   font-size: 18px;
+  height: 60px;
 }
 
-nav:hover {
-  animation: slide-left 0.5s linear;
-  animation-fill-mode: forwards;
+.nav-elements {
+  display: none;
 }
 
-@keyframes slide-left {
-  0% {
-    background-position: right;
-  }
+/* menu-icon */
 
-  100% {
-    background-position: left;
-  }
+.menu-container {
+  position: absolute;
+  top: 1%;
+  right: 0;
+  z-index: 1;
+  margin-right: 10px;
+  background: transparent;
 }
 
-.nav-elements {
-  display: none;
+.toggler {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 2;
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  opacity: 0;
 }
 
-.menu-container {
+.menu-icon {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 1;
+  width: 55px;
+  height: 50px;
+  padding: 0.8rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.line {
+  position: relative;
+  flex: none;
+  width: 100%;
+  height: 1.2px;
+  border-bottom: 2px solid var(--font-color-1);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+.line::before,
+.line::after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  top: -10px;
+  width: 100%;
+  height: 1.2px;
+  border-bottom: 2px solid var(--font-color-1);
+}
+
+.line::after {
+  top: 10px;
+}
+
+.toggler:checked + .menu-icon > .line {
+  transform: rotate(135deg);
+}
+
+.toggler:checked + .menu-icon > .line::before,
+.toggler:checked + .menu-icon > .line::after {
+  top: 0;
+  transform: rotate(90deg);
+}
+
+.toggler:checked:hover + .menu-icon > .line {
+  transform: rotate(225deg);
+}
+
+.mobile-nav-elements {
+  background: #3c3a39;
+  position: absolute;
+  right: 0;
+  top: 9%;
+  width: 100vw;
+  height: 90vh;
```

Comment:
- [ ] Please consider having the navigation menu extending the whole height of the screen as shown in the design, currently, there is a small spacing at the bottom as shown below:
![image](https://user-images.githubusercontent.com/61377322/145949060-0e40e4d9-cb99-4027-8a9e-cfde455604a6.png)